src/manifolds/GraphManifold.jl

```diff
@@ -0,0 +1,194 @@
+
+using LightGraphs
+import LightGraphs: AbstractSimpleGraph, is_directed
+import SimpleWeightedGraphs: AbstractSimpleWeightedGraph
```

Comment:
Do you actually use the functionality of `SimpleWeightedGraphs`? It looks like it was designed for different purposes.

Reply:
The edges have weights and I want to use exactly weights from edges in the `incident_log` if you habe a weighted graph, so I think I am using that exactly what it is introduced for. If you have a graph without weights, the `incident_log` assumes weights to be one.

Reply:
OK, I see. It would be a good idea to add a test with a weighted graph then, and maybe a reference to what these weights may be useful for.

Reply:
Of course. My goal (from a paper to adapt the example) is that the graph models a surface where the edges are weighted by distance and the `incident_log` models a graph laplacian, of course that needs to be documented and tested, yes – will of course do that in the `md` then.